### PR TITLE
[v2.6] Add ability to pass files in during k3s and rke2 installs

### PR DESCRIPTION
**Pull request by rancher-max**
Backport https://github.com/rancher/rancher/pull/41211 to release/v2.6

### CODE REVIEW

File: tests/validation/tests/v3_api/test_import_k3s_cluster.py [hunk 1]
```diff
@@ -168,7 +169,11 @@ def create_multiple_control_cluster():
                               'cluster_type': RANCHER_CLUSTER_TYPE,
                               'volume_size': AWS_VOLUME_SIZE,
                               'create_lb': str(K3S_CREATE_LB).lower(),
+<<<<<<< HEAD
```
Comment (slickwarren) [inline tests/validation/tests/v3_api/test_import_k3s_cluster.py:172]: some merge conflict artifacts snuck in

Comment (rancher-max) [reply tests/validation/tests/v3_api/test_import_k3s_cluster.py]: Ahh thank you, I was lazy and didn't spot this. Fixed!

**Comment by rancher-max** [comment]
Still relevant -- just needs reviews

**Review by slickwarren — APPROVED** [review]
lgtm

**Comment by rancher-max** [comment]
This change should already be present on 2.7 and 2.8 so I can just close this PR out if you all want

**Pull request closed without merging**